cirq-ionq/cirq_ionq/ionq_native_gates.py

```diff
@@ -67,22 +74,35 @@ def _circuit_diagram_info_(
 GPI = GPIGate(phi=0)
 document(
     GPI,
-    r"""The GPI gate is a single qubit gate.
-    The unitary of this gate is
-        [[0, e^{-i*2*\pi*\phi}],
-         [e^{i*2*\pi*\phi}, 0]]
-    It is driven by Rabi laser.
-    https://ionq.com/best-practices
+    r"""An instance of the single qubit GPI gate with no phase.
+
+    The unitary matrix of this gate is:
+    $$
+    \begin{bmatrix}
+      0 & 1 \\
+      1 & 0
+    \end{bmatrix}
+    $$
+
+    See [IonQ best practices](https://ionq.com/best-practices){:external}.
```

Comment:
If you don't mind changing them, these can be https://ionq.com/docs/getting-started-with-native-gates now

Reply:
Done